app/main.py

```diff
@@ -1,8 +1,48 @@
 class Car:
-    # write your code here
-    pass
+    comfort_class: int
+    clean_mark: int
+    brand: str
+
+    def __init__(self, c_class: int, c_mark: int, car_brand: str):
+        self.comfort_class = c_class
+        self.clean_mark = c_mark
+        self.brand = car_brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    distance_from_city_center: float
+    clean_power: int
+    average_rating: float
+    count_of_ratings: int
+
+    def __init__(self, distance: float, c_power: int,
+                 rating: float, count_of_votes: int):
+        self.distance_from_city_center = distance
+        self.clean_power = c_power
+        self.average_rating = rating
+        self.count_of_ratings = count_of_votes
+
+    def serve_cars(self, washes: list) -> float:
+        def calculation(car):
+            result = self.calculate_washing_price(car)
+            self.wash_single_car(car)
+            return result
```

Comment:
better call this method `serve_one_car` and write it outside `serve_cars` method and then just use it here